ModularBungalow/Kiriechem/reagent.dm

```diff
@@ -0,0 +1,67 @@
+//Kiriechem Chemicals
+/datum/reagent/medicine/bicardine //brings it back kinda
+	name = "Bicardine"
+	description = "A bruise reliever with a low overdose"
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "spicy"
+	overdose_threshold = 6
```

Comment:
6 is absurdly low for an overdose threshold

Reply:
That's by design